### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by mkhraisha**
Minor Language Changes and added clarity to the simplest order.

***
<a href="https://pr-preview.s3.amazonaws.com/mkhraisha/vc-overview/pull/4.html" title="Last updated on May 16, 2024, 6:26 PM UTC (71fc665)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-overview/4/1d697fb...mkhraisha:71fc665.html" title="Last updated on May 16, 2024, 6:26 PM UTC (71fc665)">Diff</a>

### CODE REVIEW

**Review by iherman — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -283,14 +283,14 @@ <h2>High Level View of the Specifications</h2>
         </p>
 
         <p>
-          When Verifiable Credentials are serialized in JSON, it is important to trust that the structure of a Credential may be interpreted in a consistent manner by all participants of verifiable credential ecosystem.
+          When Verifiable Credentials are serialized in JSON, it is important to trust that the structure of a Credential may be interpreted in a consistent manner by all participants of the verifiable credential ecosystem.
```
Comment (TallTed) [inline index.html:286]:
```suggestion
          When Verifiable Credentials are serialized in JSON, it is important to trust that the structure of a Credential may be interpreted in a consistent manner by all participants in the verifiable credential ecosystem.
```

File: index.html [hunk 2]
```diff
@@ -283,14 +283,14 @@ <h2>High Level View of the Specifications</h2>
...
           The [[[VC-JSON-SCHEMA]]] [[VC-JSON-SCHEMA]] defines how [[JSON-SCHEMA]] can be used for that purpose.
         </p>
 
         <p>
           Credentials can be secured using two different mechanisms: [=enveloping proofs=] or [=embedded proofs=].
           In both cases, a proof cryptographically secures a Credential (for example, using digital signatures).
-          In the enveloping case the proof wraps around the Credential, whereas enveloping proofs are included in the serialization, alongside the Credential itself.
+          In the enveloping case the proof wraps around the Credential, whereas embedded proofs are included in the serialization, alongside the Credential itself.
```
Comment (TallTed) [inline index.html:293]:
```suggestion
          In the enveloping case, the proof wraps around the Credential, whereas embedded proofs are included in the serialization, alongside the Credential itself.
```

File: index.html [hunk 3]
```diff
@@ -861,8 +861,8 @@ <h4>VC Data Integrity</h4>
...
             Some of these are metadata properties on the proof itself, like `created`, `expires`, or `domain`. 
             Others provide the necessary details on the proof generation process itself, like `cryptosuite`, `nonce` (if needed), or `verificationMethod` that usually refers to cryptographic keys.
-            The exact format, when used for Credentials, of the public keys is defined in the [[CONTROLLER-DOCUMENT]] specification, and is based on either the JWK [[RFC7517]] format or use a Multibase [[MULTIBASE]] encoding of the keys, called <a data-cite="VC-DATA-INTEGRITY#multikey">Multikey</a>.
-            Details of the key values are defined by other communities (IETF, cryptography groups, etc.) and they are very dependent on the specific cryptographic functions they operate with.
+            The exact format, when used for Credentials, of the public keys is defined in the [[CONTROLLER-DOCUMENT]] specification, and is based on either the JWK [[RFC7517]] format or a Multibase [[MULTIBASE]] encoding of the keys, called <a data-cite="VC-DATA-INTEGRITY#multikey">Multikey</a>.
```
Comment (TallTed) [inline index.html:864]:
```suggestion
            The exact format of the public keys, when used for Credentials, is defined in the [[CONTROLLER-DOCUMENT]] specification, and is based on either the JWK [[RFC7517]] format or a Multibase [[MULTIBASE]] encoding of the keys, called <a data-cite="VC-DATA-INTEGRITY#multikey">Multikey</a>.
```

File: index.html [hunk 4]
```diff
@@ -988,7 +988,7 @@ <h4>Example: the Core Example Secured with ECDSA</h4>
           </pre>
           
           <p>
-            When dereferenced, the URL `did:example:2g55q912ec3476eba2l9812ecbfe#ecdsa-public-key` should return a (public) ECDSA key in Multikey, for example:
+            When dereferenced, the URL `did:example:2g55q912ec3476eba2l9812ecbfe#ecdsa-public-key` should return a ECDSA public key in Multikey format, for example:
```
Comment (TallTed) [inline index.html:991]:
```suggestion
            When dereferenced, the URL `did:example:2g55q912ec3476eba2l9812ecbfe#ecdsa-public-key` should return an ECDSA public key in Multikey format, for example:
```

File: index.html [hunk 5]
```diff
@@ -1180,13 +1180,13 @@ <h3>Lifecycle Details</h3>
...
-          The holder presents to a verifier.
+          The holder presents the verifiable credential to a verifier.
         </li>
         <li>
-          The verifier verifies.
+          The verifier verifies the verifiable credential.
         </li>
         <li>
-          The verifier validates claims.
+          The verifier validates claims made in the verifiable credential against their business rules.
```
Comment (TallTed) [inline index.html:1189]:
```suggestion
          The verifier validates claims made in the verifiable credential against the verifier's business rules.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small language improvements

**Comment by iherman** [comment]
Comments have been adopted, not controversial, merging.

**Pull request merged**